tests/test_html.snapshot.html

```diff
@@ -385,13 +385,13 @@
 </span></div><div ><span style="font-size: 16.6pt" class="underline bold">Target Value</span><span style="font-size: 16.2pt" class="underline"> 
 </span></div><div ><span style="font-size: 10.7pt" class="italic">Total GFA of Energy Metered 
 </span></div><div ><span style="font-size: 10.7pt" class="underline italic">Data Centers</span><span style="font-size: 10.8pt" class="underline"> 
-</span></div><div ><span style="font-size: 10.7pt" class="italic">Total GFA of All Tiered Data 
-</span></div><div ><span style="font-size: 10.7pt" class="italic">Centers 
-</span></div><div ><span style="font-size: 0.6pt" class="italic"> 
-</span></div><div ><span style="font-size: 10.7pt" class="underline italic">Total Energy Used</span><span style="font-size: 10.8pt" class="underline"> 
-</span></div><div ><span style="font-size: 10.7pt" class="italic">Total IT Equipment Energy 
-</span></div><div ><span style="font-size: 10.7pt" class="italic">Used 
-</span></div><div ><span style="font-size: 0.6pt"> 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Total GFA of All Tiered Data 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Centers 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 0.6pt" class="italic"> 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="underline italic">Total Energy Used</span><span style="font-size: 10.8pt" class="underline"> 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Total IT Equipment Energy 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 10.7pt" class="italic">Used 
+</span></div><div title="Footnote 23" class="footnote"><span style="font-size: 0.6pt"> 
```

Comment:
Interesting, the algorithm seems to be thinking that some of the math in the table is a continuation of the footnote from the previous page:

> ![footnotes](https://user-images.githubusercontent.com/124687/33834505-2855cd46-de51-11e7-9295-7adc7360bc3b.png)

I guess since it's part of the table in this case, all bets are off, but it could be particularly confusing to folks who are importing since that text will actually appear as part of a footnote, rather than contiguous with the rest of the table text. That might lead them to wonder where half of their math went... Hmmm.

I guess another alternative might be to search for that horizontal rule that seems to be above the footnotes on all the pages, and use that as the demarcation point rather than the bottom third of the page. Hmm.

What do you think? We could always punt fixing this off to a separate PR too...

Reply:
Hmm, I am going to assume you're not awake in SF and I think I'll play around with doing the horizontal line detection thingy.

Reply:
Ok, I implemented the horizontal line detection in #37, let me know if it looks OK to you!